### PR TITLE
Block, Blockchain: bugfix releases (client preparation) fixing London HF at block 0 or 1 handling

### PR DESCRIPTION
**Pull request by holgerd77**
Ok, I've also included a few last minute client fixes.

### CODE REVIEW

File: packages/client/bin/cli.ts [hunk 1]
```diff
@@ -213,6 +213,22 @@ async function run() {
...
+      // move cursor back to the beginning of the input:
+      readline.moveCursor((rl as any).output, -len, 0)
+      // clear everything to the right of the cursor:
+      readline.clearLine((rl as any).output, 1)
+      // replace the original input with asterisks:
+      for (let i = 0; i < len; i++) {
+        // eslint-disable-next-line no-extra-semi
+        ;(rl as any).output.write('*')
+      }
+    })
```
Comment (holgerd77) [inline packages/client/bin/cli.ts:230]: This is a bit hacky, but due to time constraints I can't come up with a more clean solution and I would really want to have this in and not expose the private key on the screen. Feel free to directly improve yourself but please don't make this a blocker.

File: packages/client/bin/cli.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env client
+#!/usr/bin/env node
```
Comment (holgerd77) [inline packages/client/bin/cli.ts:1]: This was actually breaking the installation completely (respectively the client couldn't be used with the `ethereumjs` command. 😜

File: packages/client/bin/cli.ts [hunk 3]
```diff
@@ -310,12 +329,14 @@ async function run() {
     (args.customChainParams || args.customGenesisState || args.gethGenesis) &&
     (!(args.network === 'mainnet') || args.networkId)
   ) {
-    throw new Error('cannot specify both custom chain parameters and preset network ID')
+    console.error('cannot specify both custom chain parameters and preset network ID')
```
Comment (acolytec3) [inline packages/client/bin/cli.ts:332]: I like this pattern much better.  Awesome fix!

**Review by acolytec3 — APPROVED** [review]
Looks like it starts syncing okay on mainnet and goerli AND the `--dev` cli command still works so LGTM!

**Review by ryanio — DISMISSED** [review]

**Comment by holgerd77** [comment]
Hmm, not sure what's happening here. Ci is not running any more. 😕

Did a test with a new PR, didn't help.

Maybe some temporary GitHub thing?

**Review by ryanio — APPROVED** [review]

**Comment by holgerd77** [comment]
just published the following versions on npm:

- `@ethereumjs/block` [v3.5.1](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fblock%403.5.1)
- `@ethereumjs/blockchain` [v5.4.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fblockchain%405.4.2)

**Pull request merged**